Pipfile

```diff
@@ -41,7 +41,7 @@ hypothesis = "==6.21.6"
 protobuf = "<4.25.0,>=4.21.6"
 pytz = "==2022.2.1"
 py-ecc = "==6.0.0"
-python-dotenv = ">=0.14.5,<0.18.0"
+python-dotenv = ">=0.14.5,<0.22.0"
```

Comment:
I think we will still need the open-aea release first. Otherwise, this wont take effect.

Reply:
Yes, this is a WIP